calico_node/filesystem/sbin/check_readiness

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Check felix readiness
+if [ ! -z "$FELIX_CHECKREADINESS" ]; then
+    if [ ! /usr/bin/wget localhost:9099/readiness -O - ]; then
+        echo "Felix is not ready."
```

Comment:
It would be nice to have a consistent format for the messages users get to see.

e.g.

```
calico/node is ready
calico/node is not ready: felix is not ready
calico/node is not ready: BGP not established with 10.0.1.1 10.0.0.1
calico/node is not ready: BGP graceful restart in progress
```